pkg/oc/bootstrap/bindata.go

```diff
@@ -4738,6 +4748,9 @@ var _examplesJenkinsJenkinsEphemeralTemplateJson = []byte(`{
     }
   },
   "message": "A Jenkins service has been created in your project.  Log into Jenkins with your OpenShift account.  The tutorial at https://github.com/openshift/origin/blob/master/examples/jenkins/README.md contains more information about using this template.",
+  "labels": {
+      "app": "jenkins-ephemeral"
```

Comment:
The app label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.